cgp/hl_api.py

```diff
@@ -59,6 +62,17 @@ def evolve(
     # Main loop: -1 offset since the last loop iteration will still increase generation by one
     while pop.generation < max_generations - 1:
 
+        # reorder the genome of the parents before creating offspring
```

Comment:
i feel like this would fit better into the population class rather than the `hl_api`, don't you think so?

Reply:
What do you think about checking in `genome`? And if reorder is not possible, simply pass (maybe with a print or warning)?

Reply:
i'm not sure i understand your question? i would just remove all traces of `reorder` from this function and rather manage this decision in the population class

Reply:
Ah sorry, I thought this comment was referring to the check if reorder is possible.